templates/cinder/bin/init.sh

```diff
@@ -23,9 +23,7 @@ export DB=${DatabaseName:-"cinder"}
 export DBHOST=${DatabaseHost:?"Please specify a DatabaseHost variable."}
 export DBUSER=${DatabaseUser:-"cinder"}
 export DBPASSWORD=${DatabasePassword:?"Please specify a DatabasePassword variable."}
-export CINDERPASSWORD=${CinderPassword:?"Please specify a CinderPassword variable."}
-# TODO: nova password
-#export NOVAPASSWORD=${NovaPassword:?"Please specify a NovaPassword variable."}
+export PASSWORD=${CinderPassword:?"Please specify a CinderPassword variable."}
```

Comment:
Currently the same templates in the other service operators use PASSWORD, because the script uses the single password of its service user.
I can fix this but that makes this file only in cinder-operator looks different from the others. If we are fine with that "inconsistency" then I'll update it.

Reply:
Well, we will be changing/throwing away most of this file in the mid-term, so I'm ok being consistent with other projects for the time being.